src/System.Private.Xml/src/System/Xml/Xsl/XsltOld/Compiler.cs

```diff
@@ -799,20 +795,7 @@ internal void AddScript(string source, ScriptingLanguage lang, string ns, string
                 Hashtable typeDecls = _typeDeclsByLang[(int)langTmp];
                 if (lang == langTmp)
                 {
-                    CodeTypeDeclaration scriptClass = (CodeTypeDeclaration)typeDecls[ns];
-                    if (scriptClass == null)
-                    {
-                        scriptClass = new CodeTypeDeclaration(GenerateUniqueClassName());
-                        scriptClass.TypeAttributes = TypeAttributes.Public;
-                        typeDecls.Add(ns, scriptClass);
-                    }
-                    CodeSnippetTypeMember scriptSnippet = new CodeSnippetTypeMember(source);
-                    if (lineNumber > 0)
-                    {
-                        scriptSnippet.LinePragma = new CodeLinePragma(fileName, lineNumber);
-                        _scriptFiles.Add(fileName);
-                    }
-                    scriptClass.Members.Add(scriptSnippet);
+                    throw new PlatformNotSupportedException();  // Compiling JScript/CSharp scripts is not supported
```

Comment:
Is that another part beyond XSLT that is using CodeDom? I would really like to better understand all the usages of CodeDom in the XML stack.